prow/plugins.yaml

```diff
@@ -37,6 +37,9 @@ approve:
   - kubernetes/steering
   - kubernetes/test-infra
   implicit_self_approve: true
+- repos:
+  - kubernetes-sig-testing/frameworks
```

Comment:
You can use `kubernetes-sig-testing` to use the same approval settings on all repos in the org.